corehq/apps/app_manager/templates/app_manager/partials/releases/releases_table.html

```diff
@@ -191,24 +189,20 @@ <h4 class="panel-release-title">
         </div>
       </div><!-- .panel-body -->
       <div class="panel-footer">
-        <a class="btn btn-default track-usage-link" target="_blank"
-           data-category="Releases Page" data-action="Version Action" data-label="App Summary"
-           data-bind="attr: { href: hqImport('hqwebapp/js/initial_page_data').reverse('app_form_summary', $data.id()) }">
-          <i class="fa fa-list-alt"></i>
-          {% trans "App Summary" %}
-        </a>
+        {% if request|toggle_enabled:"VIEW_APP_CHANGES" %}
+          <a class="btn btn-default" data-bind="attr: { href: hqImport('hqwebapp/js/initial_page_data').reverse('app_form_summary_diff', $parent.previousBuildId($index()), $data.id()) }">
```

Comment:
Could you add a url function to the knockout models instead of using an `hqImport` here? The `hqImport` makes it harder to convert this page to RequireJS.

Reply:
:+1: